Language/Functions/Communication/Serial/print.adoc

```diff
@@ -94,13 +94,13 @@ void loop() {
   Serial.print("\t");
 
   Serial.print("BIN");
-  Serial.print("\t");
+  Serial.println("\t");           // carriage return after the last label 
```

Comment:
In pull request #318, I used `Serial.println("");` here, as this was consistent with what is already used in line 118 for printing a carriage return.